great_expectations/rule_based_profiler/estimators/numeric_range_estimator.py

```diff
@@ -94,7 +94,9 @@ def to_dict(self) -> dict:
         """
         Returns dictionary equivalent of this object.
         """
-        dict_obj: dict = self._configuration.to_dict()
+        dict_obj: dict = (
+            self._configuration.to_dict()  # type: ignore[union-attr] # could be None
```

Comment:
@Kilo59 Are we sure that a `None` configuration can happen?

Reply:
Yes.
```python
class NumericRangeEstimator(ABC, SerializableDictDot):
    """
    Parent class that incorporates the "get_numeric_range_estimate()" interface method, requiring all subclasses to
    implement the "_get_numeric_range_estimate()" method (for encapsulation reasons, the former calls the latter).
    """

    def __init__(
        self,
        name: str,
        configuration: Optional[Attributes] = None,
    ) -> None:
        """
        Args:
            name: the name of this estimator, which encodes the choice of the estimation algorithm: "quantiles",
            "bootstrap", "exact" (default - deterministic, incorporating entire observed value range), or "kde"
            (kernel density estimation).
            configuration: attributes needed for the estimation algorithm (subject of the inherited class) to operate.
        """
        self._name = name
        self._configuration = configuration
```

Reply:
@Kilo59 You are correct.  Sorry -- and thanks.